### PR TITLE
Keb region mapping for trial

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- adds a possibility to configure a mapping between platform region and default provider region
 for trial plan

### CODE REVIEW

File: components/kyma-environment-broker/internal/provider/azure_provider.go [hunk 1]
```diff
@@ -98,7 +105,18 @@ func (p *AzureTrialInput) Defaults() *gqlschema.ClusterConfigInput {
 	}
 }
 
-func (p *AzureTrialInput) ApplyParameters(input *gqlschema.ClusterConfigInput, params internal.ProvisioningParametersDTO) {
+func (p *AzureTrialInput) ApplyParameters(input *gqlschema.ClusterConfigInput, pp internal.ProvisioningParameters) {
+	params := pp.Parameters
+
+	// read platform region if exists
```
Comment (jasiu001) [inline components/kyma-environment-broker/internal/provider/azure_provider.go:111]: what if not exist? empty will be sent to Gardener?

**Review by jasiu001 — DISMISSED** [review]

File: components/kyma-environment-broker/internal/provider/azure_provider.go (hunk 1 above)
Comment (piotrmiskiewicz) [reply components/kyma-environment-broker/internal/provider/azure_provider.go:111]: no, the default one, see method Defaults()

**Review by jasiu001 — DISMISSED** [review]

File: components/kyma-environment-broker/README.md [hunk 2]
```diff
@@ -50,3 +50,4 @@ KEB binary allows you to override some configuration parameters. You can specify
 | **APP_VERSION_CONFIG_NAME** | Defines the name of the ConfigMap that contains Kyma versions for global accounts configuration. | None |
 | **APP_PROVISIONING_MACHINE_IMAGE** | Defines the Gardener machine image used in a provisioned node. | None |
 | **APP_PROVISIONING_MACHINE_IMAGE_VERSION** | Defines the Gardener image version used in a provisioned cluster. | None |
+| **APP_TRIAL_REGION_MAPPING_FILE_PATH** | Defines a path to the file which contains mapping between platform region and region for trial plan. | None
```
Comment (klaudiagrz) [inline components/kyma-environment-broker/README.md:53]:
```suggestion
| **APP_TRIAL_REGION_MAPPING_FILE_PATH** | Defines a path to the file which contains a mapping between the platform region and the Trial plan region. | None |
```

File: docs/kyma-environment-broker/03-01-service-description.md [hunk 3]
```diff
@@ -123,8 +123,15 @@ These are the provisioning parameters for the Trial plan that you can configure:
...
 | ---------------|-------|-------------|----------|---------------|---------------|  
 | **name** | string | Specifies the name of the Kyma Runtime. | Yes | Any string| None |  
-| **region** | int | Defines the cluster region. | No | `europe`,`us` | `europe` |  
+| **region** | string | Defines the cluster region. | No | `europe`,`us`, `asia` | Calculated from the platform region |  
 | **provider** | string | Specifies the cloud provider used during provisioning. | No | `Azure`, `GCP` | `Azure` |
  
  </details>
  </div>
+
+The region is an optional parameter. If not set, the region is calculated from platform region specified in the path:
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/03-01-service-description.md:132]:
```suggestion
The **region** parameter is optional. If not specified, the region is calculated from platform region specified in this path:
```

File: docs/kyma-environment-broker/03-01-service-description.md [hunk 4]
```diff
@@ -123,8 +123,15 @@ These are the provisioning parameters for the Trial plan that you can configure:
...
+```shell
+/oauth/{platform-region}/v2/service_instances/{instance_id}
+```
+The mapping between platform region and the provider (Azure or GCP) region is defined in a configuration file (**APP_TRIAL_REGION_MAPPING_FILE_PATH** environment variable).
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/03-01-service-description.md:136]:
```suggestion
The mapping between the platform region and the provider region (Azure or GCP) is defined in the configuration file in the **APP_TRIAL_REGION_MAPPING_FILE_PATH** environment variable.
```

**Review by klaudiagrz — APPROVED** [review]

**Review by jasiu001 — APPROVED** [review]

**Pull request merged**